### DISCUSSION TITLE
Is it posible to customize the admin sign in route "/_"?

### DISCUSSION THREAD

**Discussion by oscar-costa** [pocketbase/pocketbase#1234] [Q&A] [answered]
Hello,
Is it posible to change the admin sign in route "/_" to any custom route ex.: "/_custom_route_admin"? 
Also, is it possible to disable admin access to backend, through server console, once all the settings have been made?

**Comment by ganigeorgiev** [comment, marked as answer]
> Is it posible to change the admin sign in route "/_" to any custom route ex.: "/_custom_route_admin"?

Not directly. I'll consider in the future to make it customizable since it was requested several times already.

> Also, is it possible to disable admin access to backend, through server console, once all the settings have been made?

There is no console command for that. You can disable the admin API endpoints with a [middleware when extending PocketBase](https://pocketbase.io/docs/router/#middlewares), eg. something like:

```go
func myCustomMiddleware() echo.MiddlewareFunc {
    return func(next echo.HandlerFunc) echo.HandlerFunc {
        return func(c echo.Context) error {
        	if strings.HasPrefix(c.Path(), "/api/admins") {
        		return apis.NewForbiddenError("You are not allowed to access this resource", nil)
        	}


            return next(c)
        }
    }
}

...

app.OnBeforeServe().Add(func(e *core.ServeEvent) error {
	e.Router.Use(myCustomMiddleware())

	return nil
})
```

**Reply by alea12** [reply]
@ganigeorgiev I think the starting point to customize the admin route would be to make the `trailedAdminPath` variable configurable (we need to make sure that it wouldn't conflict with the `api` route):

https://github.com/pocketbase/pocketbase/blob/985ab1e5b79a328b516decca3d1d7dc8edfcbba0/apis/base.go#L23C7-L23C23

I noticed that your [current roadmap](https://github.com/orgs/pocketbase/projects/2/views/1) doesn't include this feature. Nevertheless, I could try making a PR for this if you feel it'd help (I need this for my own use case).

**Comment by MarkFuller1** [comment]
Has this ever been handled? For my use case I need to run two independent pocketbase instances on the same machine. Nginx needs to be able to know who to route each request to.

**Reply by ganigeorgiev** [reply]
It is implemented as part of the [refactoring](https://github.com/pocketbase/pocketbase/discussions/4355).

> For my use case I need to run two independent pocketbase instances on the same machine.

I don't see why the fixed admin route is a problem in this case (the 2 applications could also have the same `/api/*` endpoints). Without details about your nginx config I'm not sure I can help you, but in general you need to declare at least 2 different server/location blocks targeting your 2 different applications either using subdomains or subpaths. For a sample nginx config you can refer to https://pocketbase.io/docs/going-to-production/#using-reverse-proxy. If you have other questions, please open a new Q&A discussion with more details about your use case and your current nginx config.
